css-fonts/styles.css

```diff
@@ -0,0 +1,16 @@
+h1 {
+  font-family: 'Borel', cursive;
+  font-family: 'Edu TAS Beginner', cursive;
+  font-family: 'Handjet', cursive;
+  font-family: 'Noto Sans JP', sans-serif;
+  font-family: 'Work Sans', sans-serif;
+}
```

Comment:
By doing this, only 1 of these will apply. So line 6 will always beat out the other ones. So the other fonts will never actually take effect